docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -1,20 +1,18 @@
 [//]: # (title: Test code using JUnit in JVM – tutorial)
 
-This tutorial will show you how to write a simple unit test and run it with the Gradle build tool.
+This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.
 
-The example in the tutorial has the [kotlin.test](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
+The example in the tutorial has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
 
 To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
 
 ## Add dependencies
 
-1. Open a Kotlin project in IntelliJ IDEA. If you don't already have a project, [create one](jvm-get-started.md#create-an-application).
+1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project,
+   [create one](https://www.jetbrains.com/help/idea/create-your-first-kotlin-app.html#create-project).
 
-   > Specify **JUnit 5** as your test framework when creating your project.
-   >
-   {type="note"}
-
-2. Open the `build.gradle(.kts)` file and add the following dependency to the Gradle configuration. This dependency will allow you to work with `kotlin.test` and `JUnit`:
+2. Open the `build.gradle(.kts)` file and check that `testImplementation(kotlin("test"))` is present.
```

Comment:
```suggestion
2. Open the `build.gradle(.kts)` file and check that the `testImplementation`  dependency is present.
```
Visually, the wrapping of the instruction text looks a bit confusing because of alignment of the inline code bits. If you shorten `testImplementation(kotlin("test"))` so that it doesn't have Kotlin-specific syntax it looks fine. Totally up to you though!